src/Vault.sol

```diff
@@ -493,6 +496,8 @@ contract Vault {
             }
         }
 
+        require(IRegistry(registry).processWithrawal(assetAddresses, assetAmounts), "V_W: Withdrawal failed");
```

Comment:
Where is the testRevert for this require?

Reply:
Not reachable, removed the require